konnect/wrapper.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+dockerize \
+	-wait file:///kopano/ssl/konnectd-tokens-signing-key.pem \
+	-wait file:///kopano/ssl/konnectd-encryption.key \
+	-timeout 360s \
+	sleep 2
+exec konnectd serve \
```

Comment:
That seems unclean - why dockerize sleep?

Reply:
Sleep could also be at a dedicated line and not be part of the dockerize command. The problem where the sleep helps is that dockerize continues once the file has appeared. So even though the file is still being written. That in some cases lead to konnect failing to start since the certificate wasn't valid/complete.

Reply:
Mhm certificates and keys should be created atomic. Means they write to a temp file and once complete move the full file to the target which avoids half write issues. How are those files created? Needs to be fixed there instead of here.

Reply:
yes, that is indeed the better way to solve this. implemented